### PR TITLE
Avoid error of variable referenced before assignment

### PR DESCRIPTION
**Pull request by zhshuj**
There are possibility of error
 local variable 'codes' referenced before assignment
this commit is to fix it.

### CODE REVIEW

**Comment by zhshuj** [comment]
@jichenjc @SeanHQF @bjhuangr could you please review?

File: smtLayer/migrateVM.py [hunk 1]
```diff
@@ -573,6 +576,7 @@ def testMigrate(rh):
                 rh.printLn("ES", msg)
                 rh.updateResults(msgs.msg['0418'][0])
             else:
+                codes = ''
```
Comment (jichenjc) [inline smtLayer/migrateVM.py:579]: if we didn't set at line 584, this means we will likely report `empty` value
is this expected or we say something (N/A or `not able to find detailed error` ) etc?

Comment (zhshuj) [reply smtLayer/migrateVM.py:579]: > if we didn't set at line 584, this means we will likely report `empty` value is this expected or we say something (N/A or `not able to find detailed error` ) etc?


When the codes is set , the printed message is like:
`ULTMIG0420E An error occurred issuing a VMRELOCATE Move for userid COR10002. Please look up message(s): 1940 1937 in the CP Messages book for more information.']}'", 'output': ''}`

codes are `1940 1937`, which are returning CP messages error codes for VMRELOCATE command, and filled in the printed messages.

When codes is empty, the following messages are printed.

`ULTMIG0420E An error occurred issuing a VMRELOCATE Move for userid COR10002. Please look up message(s):  in the CP Messages book for more information.']}'", 'output': ''}`

If still not clear enough, I can add info about not able to find detailed error

Comment (jichenjc) [reply smtLayer/migrateVM.py:579]: ok, good for me , thanks for the detailed msg

**Review by SeanHQF — APPROVED** [review]

**Pull request merged**